controllers/lvmcluster_controller.go

```diff
@@ -65,19 +63,12 @@ type resourceManager interface {
 	ensureDeleted(*LVMClusterReconciler, context.Context, *lvmv1alpha1.LVMCluster) error
 }
 
-type ClusterType string
-
-const (
-	ClusterTypeOCP   ClusterType = "openshift"
-	ClusterTypeOther ClusterType = "other"
-)
-
 // LVMClusterReconciler reconciles a LVMCluster object
 type LVMClusterReconciler struct {
 	client.Client
 	Scheme         *runtime.Scheme
 	Log            logr.Logger
-	ClusterType    ClusterType
+	ClusterType    cluster.Type
 	SecurityClient secv1client.SecurityV1Interface
```

Comment:
Is this `SecurityClient` now used anywhere?

Reply:
No I completely removed it from the sources, and replaced it with the canonical client-go plus scheme registration